geoengine/workflow_builder/operators.py

```diff
@@ -53,6 +85,21 @@ def to_dict(self) -> Dict[str, Any]:
     def data_type(self) -> Literal['Raster', 'Vector']:
         return 'Vector'
 
+    @classmethod
+    def from_operator_dict(cls, operator_dict: Dict[str, Any]) -> 'VectorOperator':
```

Comment:
and here

Reply:
https://github.com/geo-engine/geoengine-python/pull/150/commits/b3145cd9995ab1d0a8a36f2df308923871994e67